Apps/WebClient/src/ClientApp/src/views/landing.vue

```diff
@@ -387,26 +282,44 @@
                             </div>
                         </b-col>
                     </b-row>
+                    <b-row class="py-3">
+                        <b-col>
+                            <b-button
+                                id="btnStart"
+                                :to="
+                                    isOpenRegistration
+                                        ? 'registration'
+                                        : 'registrationInfo'
+                                "
+                                role="button"
+                                class="m-2"
+                                >Register</b-button
+                            >
+                            <div class="row mobile-only"></div>
```

Comment:
instead of having a mobile-only for the size of the button, they could be in columns. If the size is desktop the columns would not be added pass 12. If it's mobile each column would be 12 causing the stacking.
https://getbootstrap.com/docs/4.1/layout/grid/#stacked-to-horizontal

Reply:
@FuriousLlama this is not for the size of the button. This approach is to move the 2nd button to the 2nd line as the mobile-only div will display only for mobile.

Reply:
btw, stacking columns seems to be a better one. I will try this :-)

Reply:
@leobcgov Couple things that I've missed. Sorry.

- add space between data set and (parentheses). e.g. Prescription Medications (Dec 2019); Lab Results (Coming Soon). Do this for all of them. 
- change "Immunization" to "Immunizations"

Reply:
@FuriousLlama I have tried putting the 2 buttons on 2 different columns but it doesn't work as the space between them is not fixed. For the desktop mockup, the Log in button should be close to the Register button. This spacing is screwed up by responsive feature of col-md or col-lg class.

Reply:
I would do something like
```
<row>
   <col class="cols-6 cols-sm-12>
   </col>
   <col class="cols-6 cols-sm-12>
   </col>
</row>
```

Reply:
it is exactly what I have in my code now. space between 2 buttons is still an issue

Reply:
I don't like the side effects of this col-sm.. when resizing screen due to the complexity of making the space between the buttons here. I'm fan of "d-flex flex-column flex-sm-row" which I believe is also a great Bootstrap approach for switching row <--> col.